docs/src/main/asciidoc/_developer-guide/How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc

```diff
@@ -16,13 +16,13 @@ The purpose of this guide is to document changes that are needed to successfully
 
 The list of {ProductName} new functionality is extensive, being the most relevant, with respect to server and application migrations:
 
-* Java EE7 - {ProductName} is a certified implementation of Java EE7, meeting both the Web and the Full profiles, and already includes support for the latest iterations of CDI (1.2) and Web Sockets (1.1).
+* Jakarta EE 8 - {ProductName} is a certified implementation of Jakarta EE 8, meeting both the Web and the Full profiles, and already includes support for the latest iterations of CDI (1.2) and Web Sockets (1.1).
```

Comment:
Full is not called "Jakarta EE 8 Full Platform" and the web is still called "Jakarta EE 8 Web Profile". We may need to change the full reference(s) to Platform.

Reply:
Good catch, updating to "Full Profile".  Platform could also work